### PR TITLE
Fix stimmy rounding pt 2

### PR DESCRIPTION
**Pull request by BrodyHughes**
Fixes BX-1541
Figma link (if any):

## What changed (plus any additional context for devs)
- couple other places in the flow where the decimals were not rounded at 6 spots

## Screen recordings / screenshots
BEFORE:
Screenshot 2024-06-26 at 11 49 35 AM
AFTER:
Screenshot 2024-06-26 at 11 49 35 AM

### CODE REVIEW

File: src/entries/popup/pages/home/Points/PointsDashboard.tsx [hunk 1]
```diff
@@ -892,6 +892,11 @@ function MyEarnings({ earnings = '0' }: { earnings?: string }) {
     decimals: 18,
     symbol: eth?.nativeAsset?.symbol,
   });
+  const earningsWithThreshold = handleSignificantDecimalsWithThreshold(
+    convertRawAmountToDecimalFormat(earnings, 18),
+    6,
+    '0.000001',
```
Comment (estebanmino) [inline src/entries/popup/pages/home/Points/PointsDashboard.tsx:898]: why with threshold and not just `handleSignificantDecimals`?

Comment (BrodyHughes) [reply src/entries/popup/pages/home/Points/PointsDashboard.tsx:898]: I saw them both and just found this easier to understand at first so i used it haha. Is there a significant difference between the two where i shouldn't use withThreshold?

**Comment by DanielSinclair** [comment]
cc @derHowie for another review

**Review by DanielSinclair — APPROVED** [review]

File: src/entries/popup/pages/home/Points/PointsDashboard.tsx (hunk 1 above)
Comment (estebanmino) [reply src/entries/popup/pages/home/Points/PointsDashboard.tsx:898]: is just one additional param that i don't think we need but won't block

**Review by estebanmino — APPROVED** [review]

**Pull request merged**